packages/firestore-compat/tools/console.build.js

```diff
@@ -57,19 +57,15 @@ const es5OutputOptions = {
   format: 'iife'
 };
 
-const PREFIX = `
-goog.module('firestore');
-exports = eval(`;
-
-const POSTFIX = ` + '${EXPORTNAME};');`;
+const POSTFIX = `window['${EXPORTNAME}']=${EXPORTNAME};`;
 
 async function build() {
   const es5Bundle = await rollup.rollup(es5InputOptions);
   const {
     output: [{ code }]
   } = await es5Bundle.generate(es5OutputOptions);
 
-  const output = `${PREFIX}${JSON.stringify(String(code))}${POSTFIX}`;
+  const output = `${String(code)}${POSTFIX}`;
```

Comment:
I think that you need a newline or at least whitespace before `${POSTFIX}` in order to match the code change in the CL. I guess the `code` always ends with a `;` so it's probably okay either way, but just in case.

Reply:
Through experimentation, I am seeing that `code` ends with both a semicolon and newline. Do you think we should add another one?

Reply:
It's probably fine, I guess there was no newline in the previous version because it was JSON.stringified.